1-Draft/RFCNNNN-ConvertFrom-Html.md

```diff
@@ -0,0 +1,185 @@
+RelationLink      Property   System.Collections.Generic.Dictionary[string,string] RelationLink {get;}
+StatusCode        Property   int StatusCode {get;}
+StatusDescription Property   string StatusDescription {get;}
+```
+
+This RFC proposes that the creation of a new CmdLet named `ConvertFrom-Html`.  This CmdLet is to be implemented into PowerShell Core and should utilize the [AngelSharp](https://github.com/AngleSharp/AngleSharp) framework for converting strings into a PSCustomObject.
+
+## Motivation
+
+As a PowerShell Core user, I can convert HTML content to objects so that I can easily work with downloaded or local HTML content.
+
+As a IT Administrator, I can call `Invoke-WebRequest` and then use `ConvertFrom-Html` to convert the `RawContent` of my Web Request to an object so that I can easily work with HTML strings/content.
+
+As a IT Administrator, I can call `Invoke-WebRequest` and then use `ConvertFrom-Html` to convert the `Content` of my Web Request to an object so that I can easily convert it to another type (json, csv, xml, etc.).
+
+As a IT Administrator, I can pipe a string into `ConvertFrom-Html` to convert it to a PSCustomObject so that I can easily convert it to another type, modify the object, and use the `ConvertTo-Html` CmdLet to convert it back to Html.
+
+## Specification
+
+The proposal is to create a new 'ConvertFrom-Html' cmdlet that will convert Html content using PowerShell core.
+
+- InputObject parameter
+  - Specifies the HTML strings to convert to Html objects. Enter a variable that contains the string, or type a command or expression that gets the string. You can also pipe a string to ConvertFrom-Html.
+  - The InputObject parameter is required, but its value can be an empty string. When the input object is an empty string, ConvertFrom-Html does not generate any output. The InputObject value cannot be $Null.
+
+### Syntax
+
+```text
+ConvertFrom-Html [-InputObject] <String> [<CommonParameters>]
+```
+
+### PARAMETERS
+
+#### -InputObject
+
+Specifies the HTML strings to convert to HTML objects.
+Enter a variable that contains the string, or type a command or expression that gets the string.
+You can also pipe a string to **ConvertFrom-Html**.
+
+The *InputObject* parameter is required, but its value can be an empty string.
+When the input object is an empty string, **ConvertFrom-Html** does not generate any output.
+The *InputObject* value cannot be $Null.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: True (ByValue)
+Accept wildcard characters: False
+```
+
+#### CommonParameters
+
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+
```

Comment:
I believe we can remove this - we haven't requiriments for them in the RFC.

Reply:
I will remove this.